ansible/roles/azure/tasks/create_adlsgen2.yml

```diff
@@ -56,26 +58,26 @@
   with_sequence: start=1 end={{ InstanceVolumesEndSequence|int }}
   when: InstanceVolumesTemp[0]|int != 0
 
-- name: Retrieve ABFSS values when specified manually 
+- name: Retrieve ABFSS values when specified manually
   set_fact:
      InstanceVolumesManual: "{{ InstanceVolumesManual +  [ item ] }}"
   loop:
     "{{ InstanceVolumesTemp }}"
   when: item.split('://')[0] == 'abfss' and  instance_volumes_input.split('|')[0].split(',') ==  ['']
 
-# This is  final list of instance volumes 
+# This is final list of instance volumes
 - name: Assign variables for autogeneration or manual for storage account creation
   set_fact:
-     InstanceVolumes: "{{ InstanceVolumesManual if  instance_volumes_input.split('|')[0].split(',') ==  [''] else InstanceVolumesAuto }}" 
+     InstanceVolumes: "{{ InstanceVolumesManual if  instance_volumes_input.split('|')[0].split(',') ==  [''] else InstanceVolumesAuto }}"
 
 - name: Update instance_volumes_preferred  in muchos.props
   lineinfile:
     path: "{{ deploy_path }}/conf/muchos.props"
     regexp: '^instance_volumes_preferred\s*=\s*|^[#]instance_volumes_preferred\s*=\s*'
     line: "instance_volumes_preferred = {{ InstanceVolumes|join(',') }}"
 
-# Not registering variable because  storage values are not visible immediately
-- name: Create ADLS Gen2 storage acount using REST API
+# Not registering variable because storage values are not visible immediately
+- name: Create ADLS Gen2 storage account using REST API
```

Comment:
Did one of the tools find this misspelling?

Reply:
I was told about [Arcolinx](https://docs.acrolinx.com/visualstudio/latest/en) by one of my colleagues and installed its plugin for VSCode (which is the IDE I use). It has a spellcheck built in, and indeed it did catch this and a few others as well. In due course, we can look for a solution which can be integrated into the CI itself.